src/modules/Login/Login.js

```diff
@@ -52,8 +54,8 @@ const Login = ({ setUser }) => {
   const loginUser = () => {
     setUser({
       username: values.username,
-      password: values.password,
     });
+    history.push('/dashboard');
```

Comment:
Forgot to mention in PR, but also changed the way LOGIN button is handled. History props are now accessed and the route to the dashboard is now passed in, instead of the Button being wrapped in a `Link` component